packages/cli/commands/create/api-sample.js

```diff
@@ -31,13 +31,17 @@ module.exports = {
         return;
       }
     }
-    const downloadSpinner = ora('Loading available API samples');
-    downloadSpinner.start();
+    const spinnies = new Spinnies();
+    spinnies.add('fetchingSampleAppsStatus', {
+      text: 'Loading available API samples',
```

Comment:
If you merge this branch with master, you should be able to make use of the new lyaml file that @miketalley added! We should get into the habit of adding any new copy to that file.